upgrading/change-log.md

```diff
@@ -7,6 +7,39 @@ heading: Version 3.15.x
 
 <p class="release-date">2020</p>
 
+<h4>Fixes</h4>
+
+- In Marimekko chart, using `yAxisValueDecimal` now works properly when `setAdaptiveYMin` and `forceYAxisValueDecimals` are enabled.
```

Comment:
This fix is valid for more than 70+ chart types with cartesian axis, not just the marimekko chart. Also the fix description can be better something link: `For cartesian axis charts, `yAxisValueDecimal` attribute now works properly when `setAdaptiveYMin` and `forceYAxisValueDecimals` attributes are enabled.`

Reply:
changed